### DISCUSSION TITLE
With the new update of Visual Studio, there are security warnings

### DISCUSSION THREAD

**Discussion by ArchieCoder** [unoplatform/uno#18852] [General]
Would it be a good idea that the Uno template have this to remove the warnings?

  <ItemGroup>
    <NuGetAuditSuppress Include="https://github.com/advisories/GHSA-3hxg-fxwm-8gf7" />
    <NuGetAuditSuppress Include="https://github.com/advisories/GHSA-5f2m-466j-3848" />
    <NuGetAuditSuppress Include="https://github.com/advisories/GHSA-x5qj-9vmx-7g6g" />
    <NuGetAuditSuppress Include="https://github.com/advisories/GHSA-xhfc-gr8f-ffwc" />
  </ItemGroup>

**Comment by DevTKSS** [comment]
The warnings are there for a reason, so would it really be good to just "remove" them? Do you speak about just silent them or what? The dependency versions throwing that warning maybe just have to be updated, so the warnings would just not appear because no problem there🤔
But that is not just the template more that the version in e.g. the uno repos would have to be updated, which will happen in the time it needs

**Reply by jeromelaban** [reply]
> Would it be a good idea that the Uno template have this to remove the warnings?

Which warnings are you talking about, specifically? If it's for System.Private.Uri, those are fixed in net9, though fixing them for net8 might cause a break in backwards compatibility. If it's refit, it will be in the next service release.

**Reply by ArchieCoder** [reply]
Oh, it looks like the 4 warning didn't make to my original post. Yes, the 3 System.Private.Uri and 1 Refit.

When I saw that the System.Private.Uri warnings are 5 years old, I told myself they are probably not important at all...

**Reply by jeromelaban** [reply]
They are, but the problem is that transitive dependencies have been included in the vulnerabilities search. The summary of the options is here: https://github.com/unoplatform/uno/issues/18019#issuecomment-2484181882.
